.github/tw-rules.yaml

```diff
@@ -0,0 +1,12 @@
+runChecks: true
+actions:
+  branch-protection-settings:
+    branches:
+      - name: main
```

Comment:
the current default branch is `master`, not `main`

Reply:
Good catch, resolved!